src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -5299,8 +5299,8 @@ private void MOV_Handler(Instruction instruction)
 
         private void ADD_Handler(Instruction instruction)
```

Comment:
For `add`, all its operands have to be pushed to stack firstly, and registers used here are transient addresses, they temporarily keep the values. 

IMHO, we are not converting all operations to stack based. For function call it probably makes more sense to utilize some registers...